### PR TITLE
Fix Send page UI

### PR DESCRIPTION
**Pull request by yuriko627**
fix html and css for the send page UI based on https://www.figma.com/file/66MKnsV8CuYraTGuLR2KXd/Wakkanay?node-id=188%3A132

### CODE REVIEW

**Comment by yuriko627** [comment]
remained task for the Send page
- https://github.com/orgs/cryptoeconomicslab/projects/22#card-32413127

File: src/components/Send.js [hunk 1]
```diff
@@ -128,112 +122,72 @@ const Send = props => {
...
-              e.preventDefault()
-              const href = `${router.route}?transfer`
-              router.push(href, href, { shallow: true })
-            }}
-          >
-            <a className="next">Next</a>
-          </div>
-        </div>
       </div>
+      <div
```
Comment (tkmct) [inline src/components/Send.js]: Please extract as PrimaryButton component

File: src/components/AddressList/AddressListItem.js [hunk 2]
```diff
@@ -94,20 +94,18 @@ export default ({
...
         height: 36px;
         padding: 8px;
       }
-      th {
-        font-weight: 500;
-      }
       .name-section,
       .address-section {
         padding: 8px;
+        color: rgba(255, 255, 255, 0.74);
```
Comment (tkmct) [inline src/components/AddressList/AddressListItem.js]: Let's use constant variables for colors.

File: src/components/AddressList/AddressListItem.js [hunk 3]
```diff
@@ -125,11 +123,11 @@ export default ({
       }
       .name {
         font-size: 18px;
-        font-weight: 300;
+        font-weight: 800;
```
Comment (tkmct) [inline src/components/AddressList/AddressListItem.js]: Let's use constant variable for font-weight value.
you can create object for this like below.
```
const FONT_WEIGHT = {
  bold: '800',
  normal: '400'
}
```

File: src/components/AddressList/AddressListItem.js [hunk 4]
```diff
@@ -125,11 +123,11 @@ export default ({
...
       }
       .address {
-        font-size: 18px;
-        font-weight: 300;
+        font-size: 14px;
```
Comment (tkmct) [inline src/components/AddressList/AddressListItem.js]: font size as well.

File: src/components/Receive.js [hunk 5]
```diff
@@ -9,71 +9,67 @@ import QRCode from 'qrcode.react'
...
-          <div className="qr-code-box">
-            <QRCode
-              className="qr-code"
-              value={props.address}
-              alt="Your QR Code"
-              size={140}
-              includeMargin={true}
-            />
-          </div>
+          <div className="address">0x0000000000</div>
```
Comment (tkmct) [inline src/components/Receive.js]: You might have to use address from redux store

File: src/components/Send.js [hunk 6]
```diff
@@ -39,87 +40,80 @@ const Send = props => {
...
-    <div>
-      <div className="send-section" id="send">
-        <div className="send-eth-title-box">
-          <div className="send-eth">Send Token</div>
-          <div className="send-icon">
-            <FontAwesomeIcon icon="sign-out-alt" />
-          </div>
+    <div className="send-section" id="send">
+      <div className="send-section-title">Send Token</div>
+      {/* <div className="balance-box">
```
Comment (tkmct) [inline src/components/Send.js]: you can just remove comments.

File: src/components/Send.js [hunk 7]
```diff
@@ -313,96 +273,43 @@ const Send = props => {
...
-        .address {
-          font-size: 15px;
-          font-weight: 500;
-          color: lightslategray;
-        }
-        .qr-code-box {
-          width: 180px;
+          align-items: center;
+          background: linear-gradient(122.3deg, #eb3959 0.21%, #c13087 93.55%);
+          border-radius: 80.7px;
```
Comment (tkmct) [inline src/components/Send.js]: Let's round up this value.

**Pull request merged**